fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -204,6 +204,12 @@ class FullyShardedDataParallel(nn.Module):
             if ``True``, reshard parameters after the forward pass. This saves
             memory but slows training. This is only relevant when resharding
             individual layers.
+        disable_reshard_on_root (bool, Optional):
+            if ``True``, reshard_after_forward will be set to False if the module is a FSDP root module.
+            It helps to improve the performance by setting disable_reshard_on_root to True if a FSDP root
+            module is the root of the entire modules.
+            For that case, we do not reshard full parameters of FSDP modules since those parameters are needed
+            immediately for the backward pass.
```

Comment:
```suggestion
            If ``True``, ``reshard_after_forward`` will be set to ``False`` if the module is a
            FSDP root module to improve performance.
            If ``False``, the performance will be lower, but it is needed because ....
            Default: True.
```

Reply:
Thanks for the update. I edited it. Can you fill in the reason after "because ...."?